### PR TITLE
Proxy containers: fixed add_repos.sh for local build

### PR DESCRIPTION
**Pull request by avshiliaev**
## What does this PR change?

Fixed add_repos.sh for local build to keep up to date. 

## GUI diff

No difference.

- [x] **DONE**

## Documentation
- No documentation needed: only internal and user invisible changes

- [x] **DONE**

## Test coverage
- No tests needed

- [x] **DONE**

## Links

Fixes #
Tracks #

- [x] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [x] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

**Review by ncounter — APPROVED** [review]
:+1: LGTM, very nice the `echo` message as well.

File: containers/proxy-httpd-image/add_repos.sh [hunk 1]
```diff
@@ -4,8 +4,10 @@ set -xe
 
 # HACK: avoid adding repos if building inside of OBS (repos come from project configuration)
 if [ -n "$1" ]; then
-    zypper addrepo http://download.opensuse.org/distribution/leap/15.3/repo/oss/ main
-    zypper addrepo http://download.opensuse.org/update/leap/15.3/sle/ updates
+    echo "WARNING: using Leap 15.4 repositories! See the add_repos.sh for more detail."
```
Comment (juliogonzalez) [inline containers/proxy-httpd-image/add_repos.sh:7]:
```suggestion
    echo "█████████████████████████████████████████████████████████████████████████████████"
    echo "█ WARNING!!! using Leap 15.4 repositories! See the add_repos.sh for more detail █"
    echo "█████████████████████████████████████████████████████████████████████████████████"
```

**Review by juliogonzalez — COMMENTED** [review]
Since the OBS logs are big, maybe we can make the warning a bit more visible :-)

File: containers/proxy-httpd-image/add_repos.sh [hunk 2]
```diff
@@ -4,8 +4,10 @@ set -xe
...
+    zypper addrepo http://download.opensuse.org/distribution/leap/15.4/repo/oss/ main
+    zypper addrepo http://download.opensuse.org/update/leap/15.4/sle/ updates
```
Comment (juliogonzalez) [inline containers/proxy-httpd-image/add_repos.sh:9]:
```suggestion
    zypper addrepo http://download.opensuse.org/update/leap/15.4/sle/ sle-updates
```

File: containers/proxy-httpd-image/add_repos.sh [hunk 3]
```diff
@@ -4,8 +4,10 @@ set -xe
 
 # HACK: avoid adding repos if building inside of OBS (repos come from project configuration)
 if [ -n "$1" ]; then
-    zypper addrepo http://download.opensuse.org/distribution/leap/15.3/repo/oss/ main
-    zypper addrepo http://download.opensuse.org/update/leap/15.3/sle/ updates
+    echo "WARNING: using Leap 15.4 repositories! See the add_repos.sh for more detail."
+    zypper addrepo http://download.opensuse.org/distribution/leap/15.4/repo/oss/ main
```
Comment (juliogonzalez) [inline containers/proxy-httpd-image/add_repos.sh:8]:
```suggestion
    zypper addrepo http://download.opensuse.org/distribution/leap/15.4/repo/oss/ main-oss
```

**Review by juliogonzalez — COMMENTED** [review]
Some other minor suggestions :-)

**Review by aaannz — APPROVED** [review]
I would expect that visible warning to be around in all of those files.

But it is IMO not worth of this roundtrip time for commit, test, push.
Correct solution I think would still be to detect OBS build (I assume there would be some specific env vars) and bail instead of plain warnings.

**Pull request closed without merging**